### PR TITLE
homework lesson2 level1

### PR DESCRIPTION
**Pull request by aMADsss**
add task1 and task2

### CODE REVIEW

File: src/main/java/students/student_aleksandrs_mads/lesson_2/homework/level_1/Task_1.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package students.student_aleksandrs_mads.lesson_2.homework.level_1;
+
+import java.util.Scanner;
+
+public class Task_1 {
+    public static void main(String[] args) {
+        Scanner scanner=new Scanner(System.in);
```
Comment (IljaMorozovs) [inline src/main/java/students/student_aleksandrs_mads/lesson_2/homework/level_1/Task_1.java:7]: Please reformat all code Ctrl+Alt+L

**Review by IljaMorozovs — CHANGES_REQUESTED** [review]

File: src/main/java/students/student_aleksandrs_mads/lesson_2/homework/level_1/Task_1.java (hunk 1 above)
Comment (antdolg1) [reply src/main/java/students/student_aleksandrs_mads/lesson_2/homework/level_1/Task_1.java:7]: Surround assignment operator with spaces here and in all other places. Also add missing spaces aroung `+ - / *`

File: src/main/java/students/student_aleksandrs_mads/lesson_2/homework/level_1/Task_2.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import java.util.Scanner;
+
+public class Task_2 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        double firstDoubleNumber = scanner.nextInt();
+        System.out.println(firstDoubleNumber);
+        double secondDoubleNumber = scanner.nextInt();
+        System.out.println(secondDoubleNumber);
+        double summ = firstDoubleNumber + secondDoubleNumber;
```
Comment (antdolg1) [inline src/main/java/students/student_aleksandrs_mads/lesson_2/homework/level_1/Task_2.java:12]: summ -> sum

**Review by antdolg1 — CHANGES_REQUESTED** [review]
Changes needed.

**Pull request merged**